tests/local_mode/test_language.py

```diff
@@ -25,3 +27,11 @@ def test_vector_non_valid_token_is_zero():
     zeros = np.zeros(actual.shape)
     # check that all cells in actual vector are zeros
     assert (actual == zeros).all() == True
+
+
+def test_language_model_download_path():
+    """Tests that downloading a model returns a valid and correct path."""
+    model_path = nlp.vocab.model_path
+    expected_model_path = os.path.join(str(Path.home()), "SyferText", "en_core_web_lg")
+    assert os.path.exists(model_path)
+    assert model_path == expected_model_path
```

Comment:
This is a nice test. Please comment each line and add empty lines to improve readability